armadillo/src/main/java/org/molgenis/armadillo/storage/ObjectMetadata.java

```diff
@@ -0,0 +1,35 @@
+package org.molgenis.armadillo.storage;
+
+import io.minio.messages.Item;
+import java.io.File;
+import java.util.Date;
+
+public class ObjectMetadata {
```

Comment:
Let's use a  `record` here since the class is immutable

Reply:
indeed.